tools/mason/MasonModify.chpl

```diff
@@ -80,13 +101,13 @@ proc masonModify(args) throws {
 proc modifyToml(add: bool, spec: string, external: bool, system: bool,
                 projectHome: string, tf="Mason.toml") throws {
 
-  const tomlPath = projectHome + "/" + tf;    
+  const tomlPath = projectHome + "/" + tf;
```

Comment:
See other nit pick. there are a few places in this file where ``join`` could be used.